### PR TITLE
Update docs to reflect extended support policy

### PR DESCRIPTION
**Pull request by Monkeyanator**
Per TOC discussion, updates official support policy to align with https://istio.io/latest/blog/2021/extended-support/.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @craigbox

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -30,10 +30,16 @@ assistance. Separately, 3rd parties and partners may offer longer-term support s
 |Type              | Support Level                                                                                                         | Quality and Recommended Use
 |------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------
 |Development Build | No support                                                                                                            | Dangerous, may not be fully reliable. Useful to experiment with.
-|Minor Release     | Support is provided until 3 months after the next minor release                                                       | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
+|Minor Release     | Support is provided until 4 and a half months after the next minor release                                            | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:33]: I believe it's 6 weeks after the second minor release following this minor release. As we do quarterly, it works out to a similar date, but best to say they have 6 weeks following the second minor release to get upgraded.

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -30,10 +30,16 @@ assistance. Separately, 3rd parties and partners may offer longer-term support s
...
 |Patch             | Same as the corresponding Minor release                                                                               | Users are encouraged to adopt patch releases as soon as they are available for a given release.
 |Security Patch    | Same as a Patch, however, it will not contain any additional code other than the security fix from the previous patch | Given the nature of security fixes, users are **strongly** encouraged to adopt security patches after release.
 
+{{< tip >}}
+Minor releases occur every 3 months and are supported for 4 and a half months after the next
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:38]: Tweak this as well based on prior comment. I would just remove the part about 4.5 months altogether.

**Review by craigbox — COMMENTED** [review]
Subsequent changes to https://istio.io/latest/docs/releases/supported-releases/ seem to have clarified this with Eric's intent, so I'll close this PR.

**Pull request closed without merging**